sdk/core/Azure.Core/perf/SerializationBenchmark/BimodalRepro.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Reflection;
+using Azure.Core.Serialization;
+using BenchmarkDotNet.Attributes;
+
+namespace Azure.Core.Perf
+{
+    [MemoryDiagnoser]
+    [InProcess]
```

Comment:
Why are you using the `InProcess` toolchain? By default BDN generates a new project, some boilerplate code and runs every benchmark in a dedicated process. It helps to avoid benchmarks affecting each other. 

Example: benchmark A allocated a lot of memory, GC has changed Gen 0 budget and the next benchmark executed within the same process (B), starts with "tuned" GC.
In .NET 8, the runtime is self-tunning (the VM is profiling the code and optimizing hot methods), this can also result in benchmarks affecting each other: every next benchmark starts with more methods already optimized.